toffy/fov_watcher.py

```diff
@@ -160,6 +167,8 @@ def on_created(self, event: FileCreatedEvent):
         """
         super().on_created(event)
 
+        print(f'{event.src_path} was just created...')
```

Comment:
I think one print message per FOV to say that it was identified/is being processed is enough. All of the callback-specific print statements are useful for debugging, but not needed while running imo